spec/hydra/pcdm/models/collection_spec.rb

```diff
@@ -26,9 +26,13 @@
         subject.members << object1
         subject.members << object2
         subject.members << collection3
+        subject.save
```

Comment:
Mike, I cleaned up the code as you suggested in your first 2 comments, as
for the save, the test will fail without it.  Trey mentioned something
about Foreign key relations not being saved, and so that's why it was
needed.  I did not quite understand what he meant.  Should I push my
corrections up?  Thank you!

On Fri, Aug 14, 2015 at 12:01 PM, Michael J. Giarlo <
notifications@github.com> wrote:

> In spec/hydra/pcdm/models/collection_spec.rb
> <https://github.com/projecthydra-labs/hydra-pcdm/pull/168#discussion_r37091087>
> :
>
> > @@ -26,9 +26,13 @@
> >          subject.members << object1
> >          subject.members << object2
> >          subject.members << collection3
> > +        subject.save
>
> Is the call to #save necessary to get IDs? I see some other specs testing
> for child_object_ids and child_collection_ids without needing a save.
> Calling #save does hurt test performance a bit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra-labs/hydra-pcdm/pull/168/files#r37091087>
> .
>